friendshipper/src/lib/components/home/ContributorLayout.svelte

```diff
@@ -221,6 +221,18 @@
 		return 'No message';
 	};
 
+	const getCommitAuthor = (commit): string => {
+		if (commit != null) {
+			if (commit.author != null) {
+				if (commit.author.name != null) {
+					return commit.author.name as string;
```

Comment:
I had to add this `as string` to make the linter happy, but have no idea why since all the types involved are fully typed. When I manually imported `Commit` and `CommitAuthor` the linter complained they weren't used, so I dunno.

Reply:
i think you want commit: Commit in the params there maybe?